includes/class-sensei-lesson.php

```diff
@@ -669,6 +669,20 @@ private function save_post_meta( $post_key = '', $post_id = 0 ) {
 			// phpcs:ignore WordPress.Security.NonceVerification
 			$new_meta_value = isset( $_POST[ $post_key ] ) ? $_POST[ $post_key ] : '';
 			$new_meta_value = Sensei_Wp_Kses::maybe_sanitize( $new_meta_value, $this->allowed_html );
+		} else if ( 'lesson_length' === $post_key ) {
+
+			$new_meta_value = isset( $_POST[ $post_key ] ) ? $_POST[ $post_key ] : '';
+
+			if ( ! is_numeric( $new_meta_value ) ) {
+
+				$new_meta_value = '-1';
```

Comment:
this is kinda weird but it is being used in other places in this function and it means "no value". Hmmm